category/index.html

```diff
@@ -0,0 +1,81 @@
+---
+title: Category
+layout: category
+---
+
+
+{% raw %}
```

Comment:
I wonder if it would be better to build all / most of this into the template file itself? e.g. to `/_includes/categories.html`

Reply:
that is a good idea, I'll work on it!